assets/src/domain/eventEditor/ui/tickets/ticketsList/actionsMenu/ticketMainMenuItem/CopyTicket.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react';
+import { pick } from 'ramda';
+import { __ } from '@wordpress/i18n';
+
+import { Copy } from '@application/ui/layout/entityActionsMenu/entityMenuItems';
+import { useTicketContext } from '@edtrHooks/index';
+import { useTicketItem } from '@edtrServices/apollo/queries';
+import { useTicketMutator } from '@edtrServices/apollo/mutations';
+import { useRelations } from '@appServices/apollo/relations';
+
+const CopyTicket: React.FC = () => {
+	const { id } = useTicketContext();
+	const ticket = useTicketItem({ id });
+	const { createEntity } = useTicketMutator();
+	const { getRelations } = useRelations();
+	const newTicket = pick(
+		['capacity', 'description', 'endDate', 'isPrimary', 'name', 'order', 'reserved', 'sold', 'startDate'],
```

Comment:
- `capacity` and `isPrimary` are properties on the datetime entity not the ticket entity 

- we don't want to copy values for 'reserved' or 'sold' since this new ticket has not yet been available for purchase, but we DO want to also copy values for all of the other ticket properties. Here's a full list of what we want to copy:

	- description
	- endDate
	- isDefault
	- isExpired
	- isFree
	- isOnSale
	- isPending
	- isRequired
	- isSoldOut <== set this to `false`
	- isTaxable
	- isTrashed
	- max
	- min
	- name
	- order
	- price
	- quantity
	- reverseCalculate
	- startDate
	- uses

Reply:
Thanks, done:

![image](https://user-images.githubusercontent.com/1477453/78045976-4726fe00-737f-11ea-89df-75ab71688d8b.png)